lib/pluginmanager/install.rb

```diff
@@ -43,9 +43,11 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    Gem.sources = Gem::SourceList.from(gemfile.gemset.sources)
```

Comment:
not sure is the right place for it, as what happen if then we've another call that aim to interact with the gem repos?, I guess we should make this happen in some kind of initialize method, run when the plugin manager gets up and running ... what do you think?